SharePointFramework/@Shared/src/models/StatusReport.ts

```diff
@@ -10,6 +10,7 @@ export class StatusReport {
   public id: number
   public created: Date
   public defaultEditFormUrl: string
+  public modified: Date
```

Comment:
I wonder if you should instead use GtLastReportDate. It's not set until the report is published, so you can fallback to modified (in case of draft).

Reply:
You can include both fields in the model actually